package.json

```diff
@@ -37,27 +37,27 @@
     "@swc/core": "~1.5.7",
     "@swc/helpers": "~0.5.11",
     "@types/jest": "^29.5.12",
-    "@types/node": "22.5.2",
-    "@typescript-eslint/eslint-plugin": "^8.4.0",
-    "@typescript-eslint/parser": "^8.4.0",
-    "babel-loader": "^9.1.3",
+    "@types/node": "22.8.6",
+    "@typescript-eslint/eslint-plugin": "^8.12.2",
+    "@typescript-eslint/parser": "^8.12.2",
+    "babel-loader": "^9.2.1",
     "eslint": "^8.57.0",
     "eslint-import-resolver-node": "^0.3.9",
     "eslint-import-resolver-typescript": "^3.6.3",
-    "eslint-plugin-import": "^2.29.1",
-    "eslint-plugin-sonarjs": "^1.0.4",
-    "husky": "^9.1.5",
+    "eslint-plugin-import": "^2.31.0",
+    "eslint-plugin-sonarjs": "^2.0.4",
+    "husky": "^9.1.6",
     "jest": "^29.7.0",
     "jest-environment-jsdom": "^29.7.0",
     "lint-staged": "^15.2.10",
-    "nx": "19.6.4",
+    "nx": "20.0.7",
```

Comment:
this one is better updated using [NX migration](https://nx.dev/nx-api/nx/documents/migrate), at least major versions. :eyes:

Reply:
yes! is better wait to do it later 😄

Reply:
[d3d6f48](https://github.com/juntossomosmais/time-out-market/pull/273/commits/d3d6f48a289c78cb227e29c9182395facbf61687)